Dockerfile

```diff
@@ -4,24 +4,70 @@ FROM certbot/dns-route53:v1.32.0
 
 ARG VERSION
 
+###
 # For a list of pre-defined annotation keys and value types see:
 # https://github.com/opencontainers/image-spec/blob/master/annotations.md
+#
 # Note: Additional labels are added by the build workflow.
-LABEL org.opencontainers.image.authors="mark.feldhousen@cisa.dhs.gov"
+###
+LABEL org.opencontainers.image.authors="vm-fusion-dev-group@trio.dhs.gov"
 LABEL org.opencontainers.image.vendor="Cybersecurity and Infrastructure Security Agency"
 
-RUN apk add --no-cache python3=3.10.8-r0
-RUN pip3 install --no-cache-dir \
-  pip==21.3.1 \
-  setuptools==60.5.0 \
-  wheel==0.37.1
+###
+# This Docker container does not use an unprivileged user because it
+# touches certbot's internal files and therefore must run as root.
+###
+
+###
+# Upgrade the system
+#
+# Note that we use apk --no-cache to avoid writing to a local cache.
+# This results in a smaller final image, at the cost of slightly
+# longer install times.
+###
+RUN apk --update --no-cache --quiet upgrade
+
+###
+# Dependencies
+#
+# Note that we use apk --no-cache to avoid writing to a local cache.
+# This results in a smaller final image, at the cost of slightly
+# longer install times.
+###
+ENV DEPS \
+    python3=3.10.8-r0
+RUN apk --no-cache --quiet add ${DEPS}
+
+###
+# Make sure pip, setuptools, and wheel are the latest versions
+#
+# Note that we use pip --no-cache-dir to avoid writing to a local
+# cache.  This results in a smaller final image, at the cost of
+# slightly longer install times.
+###
+RUN pip3 install --no-cache-dir --upgrade pip setuptools wheel
```

Comment:
I don't want to remove the existing version pinning for these packages because that is the path we _should_ be moving toward. The skeleton just needs some work to make it standard.